### PR TITLE
Add makefile

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
+
+include ./make/verbose.mk
+.DEFAULT_GOAL := help
+include ./make/help.mk
+include ./make/out.mk
```
Comment (sbose78) [inline Makefile:7]: Nothing is generated inside `/out` .
Do we need this?

File: make/out.mk [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ifndef OUT_MK
```
Comment (sbose78) [inline make/out.mk:1]: Could we remove make/.out? Don't think we need this :)

**Review by sbose78 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (alexeykazakov) [reply Makefile:7]: It's used by `check-go-fmt`.

File: make/out.mk (hunk 2 above)
Comment (alexeykazakov) [reply make/out.mk:1]: It's used by `check-go-fmt`. It creates a file inside `out` with the list of the found errors.

**Review by sbose78 — APPROVED** [review]

**Comment by alexeykazakov** [comment]
Thanks for the review @sbose78 !

**Pull request merged**